CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changed
 * Made Speck component compatible with refactored Speck library.
 * Optimized Speck component further to increase performance.
+* Update dash_bio_utils where data parsers were standardized.
```

Comment:
```suggestion
* Updated dash_bio_utils to latest version, in which data parsers were standardized.
```

Reply:
Right, I forgot about the preterite!

I'm not sure "latest version" is very fitting, because:

* strictly speaking, in a changelog context, it should be "then-latest";
* the "version" in question is a "release candidate" so even knowing it was the "then-latest version" by cross-matching release dates, it wouldn't be accurate;
* the point of the change is not to update to the latest version of utils (say, just because it's available, which would totally make sense, but it's just not the point here), it's to take advantage of these more standard data parsers.

If you suggest "in which..." why not "wherein" at this point? ;)

Reply:
>I'm not sure "latest version" is very fitting, because: ... 

Those are all valid points. Maybe instead it could say something like "Updated dash_bio_utils to version that introduced standardization to data parsers." 

>If you suggest "in which..." why not "wherein" at this point? ;)

Haha, we can have that too! Fewer characters to change :P

Reply:
Great, thanks, I'll go for "Updated dash_bio_utils to version that introduced standardization of data parsers" (because we didn't introduce standardization *to* data parsers so much, we introduced standardized data parsers, we introduced some standardization of our data parsers).